msteams-platform/overview.md

```diff
@@ -1,100 +1,55 @@
-
-* via multiple **app scopes**, such as in team channels or personally via the user's "app bar"
-* via multiple **app capabilities**, such as bots in conversation or tabs in rich web views
-
-<!-- TODO: table of capabilities, screenshots of personal scope -->
-
-Learn more about [scopes in Microsoft Teams](~/concepts/apps/apps-overview#scopes-in-microsoft-teams) and where you can surface your app.
-
-Here are the capabilities your app can leverage today. You can offer all of them in the team scope, and most of them in the personal scope.
-
-### Tabs
-
-Tabs allow team members to access your service on a dedicated canvas, within a channel or in a user's personal app space. You can leverage your existing web app to create a great tab experience within Teams.  Examples of tabs include dashboards and data visualization, documents, notes, task managers, and design canvases.
-
-Tabs in team channels allow teams to work together with the tools and data you provide, in the channel's context,and to have conversations about them.
-
-Personal tabs allow individual users to work privately with your tools and data, in their own dedicated app space.  Use these tabs to surface information catering to the individual such as personal task lists or dashboards, or provide help and support information.
-
----
-
-[Get started with tabs.](~/concepts/tabs/tabs-overview)
-
----
-
-![Example of a tab showing data, alongside a conversation about the tab data](~/assets/images/tab_example.png)
-
-### Bots
-
-Build and connect intelligent bots to interact with Microsoft Teams users naturally through chat. Bots can answer natural-language questions and perform lightweight tasks such as querying bug information or kicking off a build.​ Any bot you've created using the [Microsoft Bot Framework](https://dev.botframework.com/) can easily be made to work in Microsoft Teams.
-
-While bots can take advantage of the rich natural-language processing provided by the Bot Framework, your bot can also act as the command-line interface of your experience.  Or you can choose to make a notification-only bot, which can push information relevant to your users directly to them in a channel or direct message.
+> [!TIP]
+> Don't have Microsoft Teams? Get a free Office 365 developer subscription or activate it for your existing Office 365 account. See [Set up your Office 365 tenant](~/get-started/get-started#1-set-up-your-office-365-tenant).
 
----
-
-[Get started with bots.](~/concepts/bots/bots-overview)
-
----
-
-![Example of a bot assisting a user](~/assets/images/bot_example.png)
+## What you'll find in these topics
 
-### Compose extensions
+Are you new to the Microsoft Teams developer platform? Choose one of the following walkthroughs to quickly learn the basics of developing an app for Microsoft Teams.
```

Comment:
I think we should set the tone here saying that you will find all the relevant content that helps you extend teams to bring in your apps and services into the workspace enabling your "teams" to work on all of them right from one place. And then we can continue to say - 'Are you new to the ...'

Reply:
Done.